src/main/webapp/i18n/en/exerciseHint.json

```diff
@@ -29,6 +24,14 @@
                 "title": "Exercise Hints",
                 "tooltip": "Open hints for this exercise"
             }
+        },
+        "codeHint": {
+            "type": "Code Hint",
+            "entry": "Entry",
+            "entries": "Entries",
+            "removeEntryQuestion": "Are you sure you want to remove this entry?",
+            "noEntries": "This code hint has no solution entries.",
+            "deleted": "Deleted Code Hint with identifier {{ param }}"
```

Comment:
Is there a reason why it's sometimes upper- and sometimes lower-case?

Reply:
This may be a little bit surprising but for consistency:
In regular sentences, we would use lower case (l. 32/33) just as per usual. The entity's name in the deletion confirmation messages for exercise hints (and other entity types) is in upper case, so I decided to stay consistent there (l. 34).